README.md

```diff
@@ -0,0 +1,205 @@
+# storybook-builder-wds
+
+[Storybook builder](https://storybook.js.org/docs/web-components/builders/overview) powered by [@web/dev-server](https://modern-web.dev/docs/dev-server/overview/).
+
+# Install
+
+Install npm packages:
+
+```bash
+npm install @web/dev-server storybook-builder-wds --save-dev
+```
+
+## Setup
+
+Follow the [official storybook docs](https://storybook.js.org/) to setup storybook.
+
+You can choose Vite when asked for the builder in the init CLI.
+Later you can change it to `storybook-builder-wds` in the storybook main configuration file.
```

Comment:
This doesnt seem great to me. What does the Storybook CLI actually scaffold for you? Would it make sense to just explain the user how to scaffold it manually instead?

Reply:
CLI helps to setup a few features of storybook, I personally find it difficult yo describe them all in our docs, it's probably mostly repeating their own docs and we will have a maintenence cost to keep it up-to-date, therefore my goal here was to delegate as much as possible to their docs and tooling

but I'm also not happy with this, I think the way to solve it in the future is to add support for our builder in the storybook CLI